### ISSUE TITLE
508-defect-2 [FOCUS]: Skip to content jumps past the 4 main cards on the homepage

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25823]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a keyboard user or a screen reader user I expect for my focus to jump to the `h1` on the page "Access and manage your VA benefits and health care."

## Details

Focus jumps past the `h1` on the homepage which arguably contains the four most important cards on all of va.gov. Screen reader users who traverse the page using the `tab` key or line by line may completely miss these cards should they use skip to content.

https://user-images.githubusercontent.com/14154792/121253453-1db50c80-c877-11eb-835a-b57be5e0d620.mov

This was caught in a facilities locator usability test with an advanced screen reader user.

## Acceptance Criteria

- [ ] Skip to content sends focus to the `h1`, not "service member benefits"

## Steps to Recreate

1. Go to the homepage
2. Tab once and activate skip to content
3. Confirm focus jumps past the 4 main cards

**Comment by kelsonic** [comment]
Hey @joshkimux! 🙂 Currently that skip to content link goes to an `id="content"` on the page (assuming one exists, otherwise it just doesn't work lol) https://github.com/department-of-veterans-affairs/content-build/blob/master/src/site/includes/header.html#L110

I was thinking, what do you think if we have it just link to the first h1 tag on the page instead? Do you know if that would break on any pages across the site or if that wouldn't be a good solution for any reason?

**Comment by joshkimux** [comment]
@kelsonic great a11y eye! You're spot on when it comes to the typical expected behavior of a skip to content link. Focus usually goes to the first `h1` within the `main`. 

That being said, I can't answer your second question confidently as I'm not 100% sure if it would break behavior on other pages across the site-- that might be worth investigating into a bit more with other teams.

**Comment by 1Copenut** [comment]
@kelsonic && @joshkimux I'd be for pointing to the H1 if we can ensure it's the first element in the main content area, which I'm not super clear is always the case. In that instance, making sure we have a `main#content` available in every template would follow the GOV.UK practice: https://design-system.service.gov.uk/components/skip-link/

LMK if I've talked this into a circle or you'd like to discuss further!

**Comment by kelsonic** [comment]
@Rleahy22 will pick up this ticket, current work is on the branch `kelsonic-25823-homepage-focus`, thanks Ryan! 🔥 🚀

**Comment by brianalloyd** [comment]
@Rleahy22 shifting this to next sprint as it looks like the work is on-going.  Thanks @kelsonic for the great handoff and transition.
